Index.html

```diff
@@ -77,5 +77,170 @@ <h1>I'm Kerlos Baha.<br />Glad to see <br />you!</h1>
       </section>
       <span class="shape"> </span>
     </main>
+    <div class="space"></div>
+
+    <!-- first work page -->
+
+    <section class="repeated-section">
+      <img class="repeated-img" src="/picture/Snapshoot Portfolio.png" alt="" />
+      <p class="repeated-p1">Tonic</p>
+      <ul class="repeated-ul">
+        <li class="repeated-li1">CANOPY</li>
+        <li class="repeated-li2">Back End Dev</li>
+        <li class="repeated-li3">2015</li>
+      </ul>
+      <p class="repeated-p2">
+        A daily selection of privately personalized reads; no accounts or
+        sign-ups required.
+      </p>
+      <ul class="repeated-ul2">
+        <li class="repeated-li4">HTML</li>
+        <li class="repeated-li5">CSS</li>
+        <li class="repeated-li6">JavaScript</li>
+      </ul>
+      <button class="see-button">See project</button>
+    </section>
+
+    <div class="space-2"></div>
+    <!-- second work page -->
+    <section class="repeated-section-2">
+      <img
+        class="repeated-img"
+        src="/picture/Snapshoot Portfolio2.png"
+        alt=""
+      />
+      <p class="repeated-p1">
+        Multi-Post <br />
+        Stories
+      </p>
+      <ul class="repeated-ul">
+        <li class="repeated-li1">CANOPY</li>
+        <li class="repeated-li2">Back End Dev</li>
+        <li class="repeated-li3">2015</li>
+      </ul>
+      <p class="repeated-p2">
+        A daily selection of privately personalized reads; no accounts or
+        sign-ups required.
+      </p>
+      <ul class="repeated-ul2">
+        <li class="repeated-li4">HTML</li>
+        <li class="repeated-li5">CSS</li>
+        <li class="repeated-li6">JavaScript</li>
+      </ul>
+      <button class="see-button">See project</button>
+    </section>
```

Comment:
I believe it will be much better semantically to wrap your works section in a single `<section>` tag and wrap each project card in a `<div>` tag🙏. Typically, section tags are used to identify and separate different parts of a webpage, so you shouldn't have many section tags👍.

Reply:
Fixed.